### PR TITLE
add gitHub action close stale issues

### LINKED ISSUE #17488: Probot Stale bot should be replaced with the Close Stale Issues action
Looks like [Probot Stale](https://github.com/probot/stale) bot is not working properly lately, so it should be replaced with the [Close Stale Issues](https://github.com/marketplace/actions/close-stale-issues) GitHub Action.

The Probot Stale configuration file is located at `.github/stale.yml`.

### PR DESCRIPTION
**Pull request by NF11**
Hi, this PR could help for https://github.com/phpmyadmin/phpmyadmin/issues/17488
Signed-off-by: Nassim FARES [nassim.fares.dz@gmail.com](mailto:nassim.fares.dz@gmail.com)
Fixes https://github.com/phpmyadmin/phpmyadmin/issues/17488

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    steps:
+      - uses: actions/stale@v5
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-stale: 60
+          days-before-close: 7
+          stale-issue-label: "wontfix"
+          stale-pr-label: "wontfix"
+          stale-issue-message: >
+            This issue has been automatically marked as stale because it has not had
```
Comment (MauricioFauth) [inline .github/workflows/stale.yml:21]: I don't think it's a good idea to close stale issues right now, only pull requests.

**Review by MauricioFauth — CHANGES_REQUESTED** [review]
Thank you for your contribution.

File: .github/workflows/stale.yml (hunk 1 above)
Comment (williamdes) [reply .github/workflows/stale.yml:21]: I would even argue that I do not think we wave stale "issues"
Only stale issues with label: question

Comment (NF11) [reply .github/workflows/stale.yml:21]: I tried to follow what was on Probot Stale

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: "Mark or close stale PRs"
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
```
Comment (williamdes) [inline .github/workflows/stale.yml:5]: You should add the permissions section, see other workflows for this and documentation :)

File: .github/workflows/stale.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-stale: 60
+          days-before-close: 7
+          stale-pr-label: "wontfix"
+          stale-pr-message: >
+            This pr has been automatically marked as stale because it has not had
+            recent activity. It will be closed if no further activity occurs. Thank you
+            for your contributions.
+          exempt-issue-labels: "type: pinned,type: security"
```
Comment (williamdes) [inline .github/workflows/stale.yml:23]: We do not have a security label and it is not used actually
Is there other ways to only mark stale question labels without any response?

**Comment by divinity76** [comment]
instead of marking as "wontfix", can we instead make a dedicated "stale" label?

**Review by williamdes — CHANGES_REQUESTED** [review]
Unfortunately I do not see any use of this change. Pull-requests do not require stale management.
Could you rather help me with label:"question" as they often go stale and cause me much work to keep up to date and do manually the closing.

The message I use:
> _Could not reproduce, closing since no feedback or no way of reproducing was found.
Comment if you find a way to reproduce this issue, or want to re-open for a valid reason._

File: .github/workflows/stale.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-stale: 60
+          days-before-close: 7
+          only-pr-labels : "question"
```
Comment (williamdes) [inline .github/workflows/stale.yml:17]:
```suggestion
          only-issue-labels: "question"
```

File: .github/workflows/stale.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+          stale-pr-message: >
+            This pr has been automatically marked as stale because it has not had
+            recent activity. It will be closed if no further activity occurs. Thank you
+            for your contributions.
```
Comment (williamdes) [inline .github/workflows/stale.yml:21]: This can be removed

File: .github/workflows/stale.yml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+          close-issue-message: >
+            Could not reproduce, closing since no feedback or no way of reproducing was found.
+            Comment if you find a way to reproduce this issue, or want to re-open for a valid reason.
+          exempt-issue-labels: "type: pinned,type: security"
+          exempt-pr-labels: "type: pinned,type: security"
```
Comment (williamdes) [inline .github/workflows/stale.yml:26]:
```suggestion
```
This is not needed, but if we can post a ping message before closing it could be great

**Review by williamdes — APPROVED** [review]

**Comment by williamdes** [comment]
Merged as 0d27683e941aca2556edc225cc51f6026d3644ae

**Pull request merged**